apps/aehttp/priv/oas3.yaml

```diff
@@ -3060,6 +3062,7 @@ components:
         - difficulty
         - hashrate
         - syncing
+        - uptime
```

Comment:
strictly spoken it is not required. But it does not harm.
If the server now is required to always provide this, then you would kind of bump the version of the API, I guess.